test/test_install.py

```diff
@@ -58,6 +58,10 @@ def test_install_miniconda(tmp_path: Path) -> None:
     )
 
 
+@pytest.mark.skipif(
+    sys.version_info[:2] == (3, 7),
+    reason="Python 3.7 no longer supported by Conda",
```

Comment:
are there news on that or it is just a conda bug?
meanwhile I am running `mamba create -n python3.7 -c conda-forge -y python==3.7` and it seems to go.

Reply:
[The docs](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-python.html) currently only list Pythons 3.8 through 3.10, and [the PR that updated the docs](https://github.com/conda/conda-docs/pull/853) states that 3.7 isn't supported any more.

Reply:
wouldn't we be ok if we just use prior (23.3.0 ?) version of miniconda in our CI setup here for those matrix runs?

Reply:
datalad-installer always uses the latest Miniconda.